includes/Widgets/Stories.php

```diff
@@ -160,6 +162,24 @@ public function form( $instance ) {
 			]
 		);
 
+		$this->input(
+			[
+				'id'            => 'circle-size',
+				'name'          => 'circle_size',
+				'label'         => __( 'Circle size', 'web-stories' ),
```

Comment:
```suggestion
				'label'         => __( 'Circle Size', 'web-stories' ),
```

It's written like this elsewhere.

Let's keep it consistent so it only needs to be translated once.